cst438_gradebook/src/test/java/com/cst438/EndToEndTestSubmitGrades.java

```diff
@@ -30,29 +27,37 @@
 @SpringBootTest
 public class EndToEndTestSubmitGrades {
 
-	public static final String CHROME_DRIVER_FILE_LOCATION = "C:/chromedriver_win32/chromedriver.exe";
+	public static final String CHROME_DRIVER_FILE_LOCATION = "/Users/priya/Downloads/chromedriver-mac-x64/chromedriver";
 
 	public static final String URL = "http://localhost:3000";
 	public static final int SLEEP_DURATION = 1000; // 1 second.
 	public static final String TEST_ASSIGNMENT_NAME = "db design";
 	public static final String NEW_GRADE = "99";
+	public static final String UPDATED_ASSIGNMENT_NAME = "React Assignment";
+	public static final String ADD_ASSIGNMENT_NAME = "Database Assignment";
+
+	public static final String ADD_ASSIGNMENT_ID = "31045";
+
+	public static final String ADD_ASSIGNMENT_DATE= "10/09/2023";
+
+
+
 
 
 	@Test
 	public void addCourseTest() throws Exception {
```

Comment:
This test has much more going on an has plenty of assert statements which is good. Maybe its wise to spread some of them out to the other tests as I didn't see them. It might be redundant especially since we can see the changes live, but I presume the idea behind this is to be informed if a programs fails under bad circumstances or intense stress for the long term